.storybook/themes/news.scss

```diff
@@ -312,6 +326,11 @@
 					padding: 0 var(--global-spacing-6) 0 0,
 					display: flex,
 				),
+				subheadline: (
+					font-size: var(--global-font-size-7),
+					line-height: var(--global-line-height-9),
+					font-weight: var(--global-font-weight-7),
```

Comment:
```suggestion
					font-size: var(--global-font-size-7),
					font-weight: var(--global-font-weight-7),
					line-height: var(--global-line-height-9),
```

Reply:
Seems to be a discrepancy here that I don't see being called out?

Gazette

corecomponents subheadline: (line-height: 2rem)
![image](https://user-images.githubusercontent.com/2287238/168881836-7a2a77cb-55f3-42b7-97a5-0776730d3752.png)

vs localhost with feature pack branch for this: (line-height: 3.5rem)
![image](https://user-images.githubusercontent.com/2287238/168881984-eecd9399-2991-4aa2-b8c7-f9549f561f10.png)

Reply:
I'm going off of https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138945094/Blocks+Design+Discrepancies discussion to match the typography with the new scale. I can ask if the line height should be included in the change if we want

Reply:
Looks like the Figma token documentation for the line height is referencing the wrong token name it should be `line-height-5`

Reply:
updated!